sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -6,68 +6,99 @@
-            @sort="sortEvent"
+        <toggle-content
+          :toggled="!enrollmentsLoading && !disbursements.count"
+          message="No enrollment records found"
+        >
+          <v-data-table-server
+            v-if="disbursements?.count"
+            :headers="COESummaryHeaders"
+            :items="disbursements.results"
+            :items-length="disbursements.count"
+            :loading="enrollmentsLoading"
+            :items-per-page="DEFAULT_PAGE_LIMIT"
+            :items-per-page-options="ITEMS_PER_PAGE"
+            @update:options="paginationAndSortEvent"
           >
-            <Column field="fullName" header="Name" :sortable="true">
-              <template #body="slotProps">
-                <span>{{ slotProps.data.fullName }}</span>
-              </template>
-            </Column>
-            <Column field="studyStartPeriod" header="Study dates">
-              <template #body="slotProps">
-                <span>
-                  {{ dateOnlyLongString(slotProps.data.studyStartPeriod) }} -
-                  {{ dateOnlyLongString(slotProps.data.studyEndPeriod) }}
-                </span>
-              </template></Column
-            >
-            <Column field="applicationNumber" header="Application #"></Column>
-            <Column field="disbursementDate" header="Disbursement date">
-              <template #body="slotProps">
-                <span>
-                  {{ dateOnlyLongString(slotProps.data.disbursementDate) }}
-                </span>
-              </template></Column
-            >
-            <Column field="coeStatus" header="Status" :sortable="true">
-              <template #body="slotProps">
-                <status-chip-c-o-e :status="slotProps.data.coeStatus" />
-              </template>
-            </Column>
-            <Column field="applicationId" header="Action">
-              <template #body="slotProps">
-                <v-btn
-                  color="primary"
-                  @click="
-                    goToViewApplication(slotProps.data.disbursementScheduleId)
-                  "
-                  >View</v-btn
-                >
-              </template>
-            </Column>
-          </DataTable>
+            <template #[`item.fullName`]="{ item }">
+              <span>{{ item.fullName }}</span>
+            </template>
+            <template #[`item.studyStartDate`]="{ item }">
+              <span>
+                {{ dateOnlyLongString(item.studyStartDate) }}
+              </span>
+            </template>
+            <template #[`item.studyEndDate`]="{ item }">
+              <span>
+                {{ dateOnlyLongString(item.studyEndDate) }}
+              </span>
+            </template>
+            <template #[`item.applicationNumber`]="{ item }">
+              {{ item.applicationNumber }}
+            </template>
+            <template #[`item.offeringIntensity`]="{ item }">
+              {{ item.offeringIntensity }}
```

Comment:
Please use the formatter to display the intensity correctly (`mapOfferingIntensity`).